pagination/pagination_test.go

```diff
@@ -60,6 +60,14 @@ func TestMakeResponse(t *testing.T) {
 			total:   100,
 			expErr:  pagination.ErrCalculateOffset,
 		},
+		{
+			name:             "Page 0 must not cause underflow",
```

Comment:
Maybe there should be a test case for per page underflow, if that's possible?

Reply:
Indeed, with `PerPage`, we only multiply, and since there's no risk of multiplying with a negative number, it doesn't seem like an issue to me